src/hydra-queue-runner/build-remote.cc

```diff
@@ -367,7 +372,7 @@ void State::buildRemote(ref<Store> destStore,
             }
         }
         if (GET_PROTOCOL_MINOR(remoteVersion) >= 6) {
-            WorkerProto<DrvOutputs>::read(*localStore, from);
+          WorkerProto::Serialise<DrvOutputs>::read(*localStore, rconn);
```

Comment:
```suggestion
            WorkerProto::Serialise<DrvOutputs>::read(*localStore, rconn);
```
unintentional indent change?